### PR TITLE
Iodine frag opt

### PR DESCRIPTION
**Pull request by amcisaac**
## New Submission Checklist

- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Made a new dataset entry in the mapping table in repository `README.md`
- [x] Ready to submit!

## Compute Expansion Checklist

- [ ] Compute expansion filename matches pattern `compute*.json`; may feature a compression extension, such as `.bz2`
- [ ] QCSubmit validation passed
- [ ] Ready to submit!

### CODE REVIEW

File: submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# OpenFF multi-Br ESP Fragment Conformers v1.0
+
+## Description
+
+A dataset containing fragments of molecules from the Zinc, enamine, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments.
```
Comment (lilyminium) [inline submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md:5]:
```suggestion
A dataset containing fragments of molecules from the Zinc, Enamine 10240, Enamine 50240, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments. 
```

(just for full clarity -- there are quite a few enamine datasets!)

File: submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Description
+
+A dataset containing fragments of molecules from the Zinc, enamine, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments. 
+Fragments containing I then had protomers enumerated.
+
+For each resulting molecule, a set of up to 5 conformers were generated by:
+  * generating a set of up to 1000 conformers with a RMS cutoff of 0.5 Å using the OpenEye backend of the OpenFF toolkit
+  * applying ELF conformer selection (max 5 conformers) using OpenEye
+
+Each conformer will be converged according to the 'GAU_LOOSE' criteria.
```
Comment (lilyminium) [inline submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md:12]:
```suggestion
```

Apologies I think I myself incorrectly copy-pasted this in an earlier submission -- this actually uses the default 'GAU' convergence criteria. I'll go correct this in a different PR

File: submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## QCSubmit generation pipeline
+
+* `generate-dataset.ipynb`: This notebook shows how the dataset was prepared from the input files.
+    * Conformers were generated and selected using OpenEye
+
+
+## QCSubmit Manifest
+
+* `dataset.json.bz2`: Compressed dataset ready for submission
+* `dataset.pdf`: Visualization of dataaset molecules
```
Comment (lilyminium) [inline submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md:38]:
```suggestion
* `dataset.pdf`: Visualization of dataset molecules
```

Minor typo

**Review by lilyminium — CHANGES_REQUESTED** [review]
Thanks @amcisaac, largely looks good -- just a couple minor comments!

Non-blocking, but for the full environment, it might be helpful to also add `conda env export > full-environment-file.yaml` -- this gives largely the same information, albeit without channel information, but in the format of a file you can then (if you're on an identical architecture and with some luck) use to re-create the environment with `conda env create`.

File: submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md (hunk 1 above)
Comment (lilyminium) [reply submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md:5]: I think this description is in the dataset as generated by the notebook, as well.

File: submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md (hunk 2 above)
Comment (amcisaac) [reply submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md:12]: Ah I thought I had caught these little readme typos--looks like I didn't actually commit them. Thank you for catching it!

**Comment by lilyminium** [comment]
Thanks @amcisaac! Could you please update the description of the enamine datasets in the OptimizationDataset itself as well, please? Otherwise looks good to go!

**Review by lilyminium — APPROVED** [review]

**Comment by lilyminium** [comment]
Thanks for the fixes @amcisaac! Let me see if I can diagnose / fix CI...

**Comment by lilyminium** [comment]
I think #375 or #377 should get this going.

**Comment by amcisaac** [comment]
Thanks so much for looking into that!

**Comment by lilyminium** [comment]
FYI @amcisaac you might want to check you have git lfs installed -- I got the following message when I checked out your branch just then. I took the liberty of pushing the pointer migration just to get things running :-)

```
Encountered 1 file that should have been a pointer, but wasn't:
	submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/dataset.json.bz2
```

**Comment by lilyminium** [comment]
Whoops, the bot message reads a little more ominously from "dangerbot". (The current title format of bot comments is "Current status -- {column_where_card_should_be}". Unfortunately we have to move it ourselves manually for the time being.)

**Comment by amcisaac** [comment]
@lilyminium thank you for fixing that! I thought it was installed but looks like you're right it wasn't...weird that it didn't give me any errors when I added the file. I've installed it now so hopefully shouldn't happen again.

**Pull request merged**